src/Quartz.AspNetCore/Quartz.AspNetCore.csproj

```diff
@@ -1,25 +1,34 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-        <Description>Quartz.NET ASP.NET Core integration; $(Description)</Description>
-        <TargetFramework>netstandard2.0</TargetFramework>
-        <RootNamespace>Quartz</RootNamespace>
-        <GenerateDocumentationFile>True</GenerateDocumentationFile>
-        <Authors>Lewis Zou, Marko Lahma, Quartz.NET</Authors>
-    </PropertyGroup>
-    <ItemGroup>
-        <Compile Include="..\AssemblyInfo.cs" />
-    </ItemGroup>
+  <PropertyGroup>
+    <Description>Quartz.NET ASP.NET Core integration; $(Description)</Description>
+    <TargetFrameworks>netcoreapp2.2;netstandard2.0</TargetFrameworks>
+    <RootNamespace>Quartz</RootNamespace>
+    <GenerateDocumentationFile>True</GenerateDocumentationFile>
+    <Authors>Lewis Zou, Marko Lahma, Quartz.NET</Authors>
+  </PropertyGroup>
 
-    <ItemGroup>
-        <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.6" />
-        <PackageReference Include="Microsoft.AspNetCore.Diagnostics.HealthChecks" Version="2.2.0" />
-    </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\AssemblyInfo.cs" />
+  </ItemGroup>
 
-    <ItemGroup>
-      <ProjectReference Include="..\Quartz.Extensions.DependencyInjection\Quartz.Extensions.DependencyInjection.csproj" />
-      <ProjectReference Include="..\Quartz.Extensions.Hosting\Quartz.Extensions.Hosting.csproj" />
-      <ProjectReference Include="..\Quartz\Quartz.csproj" />
-    </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.1.1" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.2' ">
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Diagnostics.HealthChecks" Version="2.2.0" />
```

Comment:
FYI, health checks are not tied to ASP.NET Core and you could have a `Quartz.Extensions.HealthChecks` package referencing `Microsoft.Extensions.Diagnostics.HealthChecks`. `Microsoft.AspNetCore.Diagnostics.HealthChecks` only contains the ASP.NET Core middleware responsible of handling the health requests.

Reply:
I'd like to keep AspNetCore project as an umbrella for now, otherwise there will be a lot of small dlls with very specific functionality (though it's more correct).

Reply:
Makes sense 😃